### DISCUSSION TITLE
Is it possible to send form and make action on submit?

### DISCUSSION THREAD

**Discussion by error1001es** [livewire/livewire#7912] [1. Help]
```
<form method="post" action="/orders/{{ $orderId }}/accept" wire:submit.prevent="updateOrders">
    @csrf
    <button type="submit">accept</button>
</form>
```

i want to make post request for accept order in OrderController and make request to updateOrders in ControlOrdersComponent

what is the best way to move the logic, since some actions from ControlOrdersComponent will be reused in other components, I thought that moving the logic to the OrderController would be a great idea, at least for my application

sometimes I need data in a form by type

```
<select name="user_id>
<input name="total">
```

in accept or another methods with order

can i use this way?

```
class OrderController {
    public function reject($orderId) {
    }
}

class ControlOrdersComponent {
    private $orderController;

    public function __construct() {
        $this->orderController = new OrderController();
    }

    public function reject($orderId) {
        $this->orderController->reject($orderId);
    }
}
```

**Comment by calebporzio** [comment]
I wouldn't try submitting a form to a controller AND a component.

Just stick to one and work out what you need for reusability on the backend in other ways.

**Comment by error1001es** [comment]
i have 5 different methods for order and 2-3 of them I will reuse in other places

Is this method suitable for reuse?
```
class OrderController {
     public function reject($orderId) {
     }
}

class ControlOrderComponent {
     private $orderController;

     public function __construct() {
         $this->orderController = new OrderController();
     }

     public function reject($orderId) {
         $this->orderController->reject($orderId);
     }
}
```
